### PR TITLE
Bias initialization

### PR DESCRIPTION
**Pull request by hazirbas**
Since batch normalization already is trained for shift, conv bias parameters are not necessary. In case of instance normalization, one should still use bias but initialize to zero. Otherwise, default one initializes bias with uniform distribution between -stdv, +stdv [source](http://pytorch.org/docs/master/_modules/torch/nn/modules/conv.html#Conv1d)

### CODE REVIEW

**Comment by junyanz** [comment]
Thanks a lot, @hazirbas. This is a great catch.
